parquet2hive/parquet2hivelib.py

```diff
@@ -27,7 +27,8 @@ def get_bash_cmd(dataset, success_only = False, recent_versions = None, version
     bash_cmd, versions_loaded = "", 0
     for (version_prefix, dataset_name, version) in versions:
         sample, success_exists = "", False
-        for key in bucket.objects.filter(Prefix=version_prefix):
+        keys = sorted(bucket.objects.filter(Prefix=version_prefix), key = lambda obj : obj.last_modified, reverse = True)
```

Comment:
Yeah, the network requests can be slow at times, depending on how many object headers have to made. Have you benchmarked it with the main summary dataset as well? I am asking because when we implemented a similar functionality in the `python_moztelemetry` library we had to parallelize the code to get some good performances.